doc/ruleSets/PER.rst

```diff
@@ -4,6 +4,14 @@ Rule set ``@PER``
 
 Alias for the PER-CS rules. It is recommended you use ``@PER-CS2.0`` instead.
 
+Warning
+-------
+
```

Comment:
Not sure if we want to deprecate it 

We had an idea (not sure if officially incorporated or not yet) that:
We have PER-CS set with versions, so one can lock into specific version like v1 or V2, and we have set without mentioning the version, acting as "newest".

(Then , maybe it should be called PER-CS and not simply PER indeed)

Wdyt ?

Reply:
It was already deprecated in #6707 during rulesets' refactor. And yes, it's related to incomplete name,however `@PER-CS` was not introduced after [discussion](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6707#discussion_r1199507132). So this change is only an introduction of new deprecation handling 🙂.

Reply:
Overall we do not have locking mechanism for rulesets and any time someone will create new fixer that is fulfilling spec of ruleset (egg Symfony), the ruleset will change.

Even with PER CS V2, V2 is version of spec, not of ruleset, so as soon as we will have more rules implemented, ruleset will start behaving differently, even if someone is pinning v2 spec.

For that, I'm in favour of having the "newest" alias.

Reply:
Not a problem for me, but this is not a concern of this PR 🙂. This ruleset was already deprecated, docs are just updated to match current state. I won't introduce new alias in this PR, I can do it in separate one.

Reply:
TODO: create a PER-CS ruleset that aliases the "newest" PER-CS set

Reply:
-=> #7297